lang/en.json

```diff
@@ -0,0 +1,67 @@
+  "Maneuvers.Analyze.Name": "Analyze",
+  "Maneuvers.Betray.Desc": "Make a Skill Check. If you succeed, Target Foe is OFF-GUARD until your next Turn.",
+  "Maneuvers.Betray.Name": "Betray",
+  "Maneuvers.Daunt.Desc": "When you use a Move Action, you may make a Skill Check. If you succeed, you may pass through Squares Occupied by Foes during that Action.",
+  "Maneuvers.Daunt.Name": "Daunt",
+  "Maneuvers.Fade.Desc": "Make a Skill Check. If you succeed, Foes cannot Target you until your next Turn.",
+  "Maneuvers.Fade.Name": "Fade",
+  "Maneuvers.Feint.Desc": "Make a Skill Check. If you succeed, +2 A to your next Action. 1 Stack.",
+  "Maneuvers.Feint.Name": "Feint",
+  "Maneuvers.Focus.Desc": "Make a Skill Check. If you succeed, +1 Healing or Damage Die to your next Action. 1 Stack.",
+  "Maneuvers.Focus.Name": "Focus",
+  "Maneuvers.Guide.Desc": "Make a Skill Check. If you succeed, 1 Other Hero gains Advantage on his or her next Roll. 1 Stack.",
+  "Maneuvers.Guide.Name": "Guide",
+  "Maneuvers.Optimize.Desc": "Make a Skill Check. If you succeed, you may convert A into P at a ratio of 2:1 or vice versa. Optimize and its Effect is removed if you become FALLEN or at the end of the Encounter.",
+  "Maneuvers.Optimize.Name": "Optimize",
+  "Maneuvers.Refresh.Desc": "Make a Skill Check. If you succeed, heal [Tier]+4 HP.",
+  "Maneuvers.Refresh.Name": "Refresh",
+  "Maneuvers.Spring.Desc": "When you use a Shift Action while adjacent to a Foe, make a Skill Check. If you succeed, +2 Squares to that Action.",
+  "Maneuvers.Spring.Name": "Spring",
+  "Maneuvers.Study.Desc": "Make a Skill Check. If you succeed, you may look at 1 Foe's Hex Grid, Default Characteristics, or Special Abilities Table.",
+  "Maneuvers.Study.Name": "Study",
+  "Maneuvers.StudyVariant.Desc": "Make a Skill Check. If you succeed, you may look at 1 Foe's Foe Card.",
+  "Maneuvers.StudyVariant.Name": "Study (Variant)",
+  "Maneuvers.Tumble.Desc": "Move 1-3 Squares. Can move through any Combatant.",
+  "Maneuvers.Tumble.Name": "Tumble",
+  "Maneuvers.Vault.Desc": "Make a Skill Check. If you succeed, your next Move Action can be used to traverse terrain vertically. Your Move Action must end on a Square you can Occupy.",
+  "Maneuvers.Vault.Name": "Vault",
+  "Maneuvers.Weather.Desc": "When you enter a Square with a Local Field Effect, you may make a Skill Check. If you succeed, ignore the Local Field Effect.",
+  "Maneuvers.Weather.Name": "Weather",
+  "Skills.Acrobatics": "Acrobatics",
+  "Skills.Athletics": "Athletics",
+  "Skills.Endurance": "Endurance",
+  "Skills.FastTalk": "Fast-Talk",
+  "Skills.Focus": "Focus",
+  "Skills.Insight": "Insight",
+  "Skills.Intimidate": "Intimidate",
+  "Skills.Knowledge": "Knowledge",
+  "Skills.Leadership": "Leadership",
+  "Skills.Mend": "Mend",
+  "Skills.SleightOfHand": "Sleight of Hand",
+  "Skills.Stealth": "Stealth",
+  "Equipment.Sword": "Sword [$1H]",
```

Comment:
This is probably over-engineering, and I need to see if the built-in translation system supports passing arguments to fill in `$1`, `$2`, etc. It makes for a more extensible system when position matters but the value in that position does not. It would also save some lines since otherwise we would need things like `+1 Accuracy`, `+2 Accuracy`, etc.

Reply:
Looks like it supports taking in an object and doing replacements based on the names of the attributes
![image](https://user-images.githubusercontent.com/1091716/81251927-871e7800-8ff2-11ea-9b07-d44baa100bfb.png)

Reply:
We could probably build a string of `"+{bonus} {attribute}"` or something